pkg/test/framework/suite.go

```diff
@@ -264,6 +266,20 @@ func (s *suiteImpl) RequireMultiPrimary() Suite {
 	return s
 }
 
+func (s *suiteImpl) SkipConfigClusterTopology() Suite {
```

Comment:
One small naming suggestion, but otherwise LGTM.

I think the following would be easier to understand:
```
func (s *suiteImpl) SkipExternalControlPlaneTopology() Suite {
	fn := func(ctx resource.Context) error {
		for _, c := range ctx.AllClusters() {
			if c.IsExternalControlPlane() {
				s.Skip(fmt.Sprintf("Cluster %s is an external control plane, we can't run external control plane topology",
					c.Name()))
			}
		}
		return nil
	}
	s.requireFns = append(s.requireFns, fn)
	return s
}
```

Reply:
I agree, updated the name.